aibolit/metrics/cognitiveC/cognitive_c.py

```diff
@@ -44,6 +44,21 @@ def traverse_childs(self, block: Any, nested_level: int) -> None:
         for each_child in block.children:
             self.get_complexity(each_child, nested_level)
 
+    def __if_stat(self, expr, nested_level: int) -> None:
+        '''function to work with IfStatement block'''
+        self.get_complexity(expr.condition, 0)
+        if expr.then_statement is not None:
+            self.complexity += nested_level + 1
+            self.get_complexity(expr.then_statement, nested_level + 1)
+
+        if expr.else_statement is not None:
+            if isinstance(expr.else_statement, IfStatement):
+                self.complexity -= nested_level
+                self.__if_stat(expr.else_statement, nested_level)
+            else:
+                self.complexity += 1
+                self.get_complexity(expr.else_statement, nested_level + 1)
+
     def increment_logical_operators(self, block: BinaryOperation, prev_operator: str) -> None:
```

Comment:
all internal function of the class must have prefix `_`

Reply:
done